src/components/applications/ApplicationDetails.js

```diff
@@ -47,14 +49,57 @@ export const ApplicationDetails = ({application, isHR, reload}) => {
     }
 
     const nextStage = () => {
+        if ( nextStageRequiresDevs() ) {
+            const devMails = []
+            Swal.fire({
+                title: "We need more info",
+                icon: "warning",
+                text: 'Next stage of this appliaction requires people with technical knowledge. Please, insert their mails here, splitting them with commas (,)',
+                input: "text",
+                preConfirm: (devsMails) => {
+                    if (devsMails) {
+                        devsMails.split(",")
+                            .map(mail => mail.trim())
+                            .forEach(mail => devMails.push(mail))
+                        devMails.forEach(mail => {
+                            if(!validateEmail(mail)) {
+                                Swal.showValidationMessage(`${mail} is not a valid email`)
+                                devMails.length = 0
+                            } 
+                        })
```

Comment:
```suggestion
                        const mailsToCheck = devsMails.split(",")
                            .map(mail => mail.trim())
                        const invalidMail = mailsToCheck
                            .find(mail => !validateEmail(mail))
                        if (invalidMail == null)
                            devMails = mailsToCheck
```
Taka propozycja, `devMails.length` jest dla mnie za potężne, przerobiłbym to na zwykłego let z pustą tablicą

Reply:
Wierz lub nie, ale `.length = 0` jest legitnym sposobem na czyszczenie tablicy w JSie XD
A tak na serio, pomyślę, bo to jednak wszystko w Swalu jest robione i może być nieciekawie.